addons/mail/static/src/components/attachment_viewer/attachment_viewer.scss

```diff
@@ -49,11 +49,21 @@
     color: lighten($gray-400, 15%);
 }
 
-.o_AttachmentViewer_toolbarButton.o_disabled {
-    filter: brightness(1.3);
+.o_AttachmentViewer_toolbarButton {
+    background-color: var(--AttachmentViewer_toolbarButton-background-color, #{$o-gray-800});
+    color: #fff;
```

Comment:
Not sure to understand, background-color can change but not the font color?

Reply:
These buttons must keep the same color for both modes (light and dark).
But as the grayscale changes in dark mode, we must change the color with the css variable set here :
https://github.com/odoo/enterprise/pull/31128/commits/70f7115ea383fa55f4ef7ddca890260bb834d749#diff-220c61aebcd47357565675b0aa99b1093427a794a1b6aad3690906ec3ff24a46R6